resources/angrychef/msi/source.wxs.erb

```diff
@@ -47,11 +47,6 @@
               </Component>
             </Directory>
             <Directory Id="PSMODULES" Name="modules" >
-              <Directory Id="PSMODULESCHEF" Name="chef" >
-                <Component Id="ChefPSModuleFile" Guid="{DF0E99BF-1877-4099-85A4-067C84A743F2}" >
-                  <File Id="chef.psm1" Source="$(var.ProjectSourceDir)\embedded\apps\chef\distro\powershell\chef\chef.psm1" KeyPath="yes" />
-                </Component>
-              </Directory>
```

Comment:
There is no embedded apps directory any more :).  I moved the responsibility of deploying these files into the software definition of chef.